bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/scheduling/Strand.java

```diff
@@ -52,6 +53,7 @@
     public Object[] frames;
     public int resumeIndex;
     public Object returnValue;
```

Comment:
why do we have it as public? I think it is better to have it as private and access it via a method call. It will hide the fields in the object and future modification would be easier

Reply:
We use these fields in codegen as well. So we need to have them public. If not codegen side need to change with private methods but that will be a considerable change.